src/App.js

```diff
@@ -1,19 +1,48 @@
 import { useState, useEffect } from 'react'
 import { BrowserRouter as Router, Route } from 'react-router-dom'
 import Landing from './components/Landing'
+import Navigation from './components/Navigation'
+import Footer from './components/Footer';
+import Contact from './components/Contact';
+import UserProfile from './components/UserProfile';
+import SignUp from './components/SignUp';
 
 function App() {
   return (
     <Router>
       <div className='container'>
-        {/* Navigation Menu */}
+        {/* Navigation Menu compent prop added to each page */}
         {/* Landing Page */}
-        <Landing />
+        
+        <Route path= '/' component={Navigation}/>
+        <Route path='/'
+        exact
+        render={(props) => (
+          <>
+          <Landing/>
+          <Route path= '/' component={Footer}/>
+          </>
+        )}
+        />
+        <Route path= '/Landing' component={Landing}/>
+        <Route path= '/Landing' component={Footer}/>
+        <Route path= '/Landing' component={Navigation}/>
         {/* Sign Up Page */}
+        <Route path= '/SignUp' component={SignUp}/>
+        <Route path= '/SignUp' component={Footer}/>
+        <Route path= '/SignUp' component={Navigation}/>
         {/* Login Page */}
         {/* Dashboard Page */}
         {/* Profile Settings Page */}
+        <Route path= '/UserProfile' component={UserProfile}/>
+        <Route path= '/UserProfile' component={Navigation}/>
+        <Route path= '/UserProfile' component={Footer}/>
         {/* Add Animal Page */}
+        {/* About Page */}
+        {/* Contact Us Page */}
+        <Route path= '/Contact' component={Contact}/>
+        <Route path= '/Contact' component={Navigation}/>
+        <Route path= '/Contact' component={Footer}/>
```

Comment:
It would be cleaner to import the different components in each of the individual components then to have multiple different Routes per path. For example, instead of 

<Route path= '/Landing' component={Footer}/>
<Route path= '/Landing' component={Navigation}/>

you would import  "Footer" and "Navigation" in "Landing.js" and use it there.

Reply:
It's also probably because some of the pages/routes don't exist yet, but when I, for example, click "Contact" from the Landing page, and then try to navigate back to the Landing page, the url path becomes "/Contact#Home". Should it be "/Home" instead? Also, it does not navigate back to the Home page. The web page stays on Contact Us.

Reply:
I have another PR to fix the # - I connected all the pages in the nabber links which removes this.